gui_agents/s3/core/engine.py

```
@@ -443,3 +443,95 @@ def generate(self, messages, temperature=0.0, max_new_tokens=None, **kwargs):
         )
+
+
+class LMMEngineDeepSeek(LMMEngine):
+    def __init__(
+        self,
+        base_url=None,
+        api_key=None,
+        model=None,
+        rate_limit=-1,
+        temperature=None,
+        **kwargs,
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)